Codeinwp/neve
# Fix/issue 1652

### Summary
Improve Dynamic CSS generation based on context ( frontend/gutenberg ) and fix the issues with inner content blocks width, refi #1652 
The blocks which allow inner content blocks ( cover/group/section) should have the width of the content  
### Will affect the visual aspect of the product 
Yes
 

### Test instructions
- Create sample paragraph blocks and add one cover block with multiple p blocks inside of it. 
- The width of the inner blocks ( inside cover ) should have the same width as the content. 
- Check this on various contexts based on: 
1. Customizer Layout - content/full-width
2. Customizer content width:  100%, < 100%, with sidebar, without sidebar. 
3. Individual pages, with meta set for content/full-width, content width, with/without sidebar. 
 
Closes #1652
